### ISSUE TITLE
Add mipmapping to renderer for downscaling in hyprexpo

### ISSUE THREAD

**Issue by KZDKM** [hyprwm/Hyprland#5569]
### Description

For both hyprexpo and my plugin [Hyprspace](https://github.com/KZDKM/Hyprspace), the textures of window and layer surfaces in overviews are squished down into a smaller size and rendered as is with `GL_LINEAR` filter. This results in windows and layers in overview being very pixelated, sharp and wobbly. 

Is it possible implement an additional field in `m_RenderData` struct (just like `useNearestNeighbor`) to tell `renderTexture` calls to create a mipmap or use other methods to downscale the surface textures?

**Comment by vaxerski** [comment]
I've tried (there is scale) but it doesn't scale the shaders so it looks wrong, plus many calculations tend to break.

TL;DR: tried, dunno how. Feel free to make a MR

**Comment by vaxerski** [comment]
Most windows should have unset (aka default) uv after calculations, though. It's only when a window is clipped/stretched

**Comment by KZDKM** [comment]
Might be related: https://gitlab.freedesktop.org/wlroots/wlroots/-/issues/3814

**Comment by vaxerski** [comment]
Hello there!

This issue has been closed, as we are moving from issues to discussions. More about this can be found [here](https://wiki.hyprland.org/Contributing-and-Debugging/Issue-Guidelines/) and in #9854.

Firstly, please make sure this issue is still relevant in latest Hyprland. If so, we ask you to open a [discussion](https://github.com/hyprwm/Hyprland/discussions) (please read the discussion guidelines first, in the pinned post)

Thank you for your understanding!
